src/compute/src/rpc/service/exchange_service.rs

```diff
@@ -119,13 +129,28 @@ impl ExchangeServiceImpl {
                             }),
                             Err(e) => Err(e.to_grpc_status()),
                         };
+
+                        let bytes = match res.as_ref() {
+                            Ok(msg) => Message::get_encoded_len(msg),
+                            Err(_) => 0,
+                        };
+
                         let _ = match tx.send(res).await.map_err(|e| {
                             RwError::from(ErrorCode::InternalError(format!(
                                 "failed to send stream data: {}",
                                 e
                             )))
                         }) {
-                            Ok(_) => Ok(()),
+                            Ok(_) => {
+                                metrics
+                                    .stream_exchange_bytes
+                                    .with_label_values(&[
+                                        &up_down_ids.0.to_string(),
```

Comment:
We should store the string in advance, instead of calling `to_string` every time we get a new chunk.